python/ray/data/dataset.py

```diff
@@ -4742,7 +4742,7 @@ def serialize_lineage(self) -> bytes:
             .. testoutput::
 
                 Dataset(
-                   num_blocks=16,
+                   num_blocks=8,
```

Comment:
CC: @bveeramani the output of this test is curated based on the host machine type; is there a way to make it more generic (so that they can run reliably regardless of how many core the machine has). Thankkks

Reply:
You can replace it with ellipsis (`...`)

Reply:
beautiful, thank you!